docker-images/kafka/Dockerfile

```diff
@@ -17,15 +18,12 @@ RUN useradd -r -m -u 1001 -g 0 kafka
 ENV KAFKA_HOME=/opt/kafka
 ENV KAFKA_VERSION=${KAFKA_VERSION}
 ENV STRIMZI_VERSION=${strimzi_version}
-ENV SCALA_VERSION=2.12
-ENV KAFKA_CHECKSUM="${KAFKA_SHA512}  kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz"
-
-RUN curl -O https://archive.apache.org/dist/kafka/${KAFKA_VERSION}/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz \
-    && echo $KAFKA_CHECKSUM > kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz.sha512 \
-    && sha512sum --check kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz.sha512 \
-    && mkdir $KAFKA_HOME \
-    && tar xvfz kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz -C $KAFKA_HOME --strip-components=1 \
-    && rm -f kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz*
+
+COPY $KAFKA_SOURCE_FILENAME .
+
+RUN mkdir $KAFKA_HOME \
```

Comment:
You seem to first copy the archive and then unpack it and delete the archive. That IMHO results in two layers in the Docker image - one of them being useless. Why not unpack it outside and just copy it to the Docker image?

Reply:
This is a good idea. Would it make sense to do the same exercise with the Kafka Exporter? We could add the version, checksum and URL to the `kafka-versions.yaml` file? Or is that a separate PR?

Reply:
Ok the script now unpacks the tar outside the docker file (in the tmp directory) and then copies the file into the container.